components/InfoWindow.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <v-card
+    max-width="300"
+    class="pt-5 px-0 pb-0">
+    <v-img
+      max-width="300"
+      height="169px"
+      class="white--text"
+      :src="`${marker.thumbnail_url}?authorization=${token}`"
+    >
+    </v-img>
+
+    <v-card-text class="pa-0 text-center black--text overline">
+      {{marker.name}}
+      <nuxt-link to="cameras/https-bhgfx">
```

Comment:
The last small change and it is fine for me to merge: You have the camera id stored in the marker.
```suggestion
      <nuxt-link :to="`cameras/${marker.id}`">
```